website/screens/components/textarea/usage/TextareaUsagePage.tsx

```diff
@@ -9,8 +9,8 @@ const sections = [
     content: (
       <DxcList>
         <DxcText>
-          Use the texarea when you need users to enter a text longer than a
-          single line.
+          Use the texarea when users need to enter a text longer than a single
```

Comment:
About this rephrasing, it should be "...to enter a text longer..." or "...to enter text longer..."? 🤔

Reply:
I think better without "a" because it is repeated later.

Reply:
Done